tests/plugins/widget/manual/outline.md

```diff
@@ -0,0 +1,13 @@
+@bender-ui: collapsed
+@bender-tags: 4.9.0, 1535, feature
+@bender-ckeditor-plugins: wysiwygarea,toolbar,undo,elementspath,clipboard,floatingspace,sourcearea,htmlwriter,image2,format,blockquote
+
+## For each editor instance:
+
+1. Hover on widget.
+1. Focus widget.
+1. Focus widget editable.
+
+### Expected:
+
+Widget outline on hover and on focus has new styling. 
```

Comment:
I'm afraid you have to be more explicit here, I feel like not everyone will know what the change in styling is (compared to previous version).

Thus it make sense to put an information that focus indicator is expected to have orange color, while unexpected is a yellow color. And also info on thickness.